packages/prisma/zod/custom/eventtype.ts

```diff
@@ -1,15 +1,17 @@
 import { _EventTypeModel } from "../eventtype";
 
 export const createEventTypeInput = _EventTypeModel
-    .pick({
-        title: true,
-        slug: true,
-        description: true,
-        length: true,
-        teamId: true,
-        schedulingType: true,
-    })
-    .refine((data) => (data.teamId ? data.teamId && data.schedulingType : true), {
-        path: ["schedulingType"],
-        message: "You must select a scheduling type for team events",
-    });
\ No newline at end of file
+  .pick({
+    title: true,
+    slug: true,
+    description: true,
+    length: true,
+    teamId: true,
+    schedulingType: true,
+    hidden: true,
+  })
+  .partial({ hidden: true })
```

Comment:
Why is this needed?

Reply:
@zomars 
Looks like when creating event types via API, hidden field it's omitted sometimes. With these we flag hidden as an optional field, and if it's missing on the data before creating it, it will default to create eventType with default false value.